src/showcase/app/shared/component-viewer/example-viewer/example-viewer.component.ts

```diff
@@ -43,5 +62,70 @@ export class ExampleViewerComponent implements OnInit {
     this.ts = this._htmlLoader.with(this._route).fromExamples(this.name, 'ts').observe();
 
     this.scss = this._htmlLoader.with(this._route).fromExamples(this.name, 'scss').observe();
+
+    this._htmlLoader
```

Comment:
This could be refactored to:
```ts
    const exampleContents = ['ts', 'html', 'scss'].map((type: 'ts' | 'html' | 'scss') =>
      this._htmlLoader
        .with(this._route)
        .fromSourceExamples(this.name, type)
        .observe()
        .pipe(map((content) => ({ name: `${this.name}${type}`, content })))
    );
    forkJoin(exampleContents).subscribe((result) => this.createStackblitzForm(result));
```